x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceNamedWriteablesProvider.java

```diff
@@ -704,6 +705,13 @@ private static void addInferenceResultsNamedWriteables(List<NamedWriteableRegist
                 StreamingUnifiedChatCompletionResults.Results::new
             )
         );
+        namedWriteables.add(
```

Comment:
This isn't really necessary. It's kind of a hack that `AuthorizationResponseEntity` needs to implement `InferenceServiceResults`. We need to do that to make it work with the sender framework we have. The previous response entity wasn't registered. I figured it'd be more clear if we registered it 🤷‍♂️